test/e2e/service.go

```diff
@@ -266,10 +270,6 @@ var _ = Describe("Services", func() {
 				CreateExternalLoadBalancer: true,
 			},
 		}
-
-		By("cleaning up previous service " + serviceName + " from namespace " + ns)
```

Comment:
I think that some developers start a k8s cluster and run tests multiple times.

We shouldn't remove the cleanup until the namespaces are unique.